projects/plugins/jetpack/extensions/blocks/blogroll/blogroll-item/editor.scss

```diff
@@ -1,5 +1,12 @@
-/**
- * Editor styles for Blogroll Item
- */
+.wp-block-jetpack-blogroll-item {
 
-.wp-block-jetpack-blogroll-item { }
+	font-size: 14px;
+	font-style: normal;
+	font-weight: 400;
+
+	.wp-block-columns {
+		padding: 20px 8px;
+		margin: 0;
+	}
+
+}
```

Comment:
If someone else can confirm they're also seeing the blockroll item icon placeholder image overflowing in the editor, we can add this styling to set it to 48px width/height:

<img width="377" alt="CleanShot 2023-09-06 at 16 19 17@2x" src="https://github.com/Automattic/jetpack/assets/10482592/74d8eca2-a65a-44d3-b947-f487fc100516">


```
.components-placeholder.block-editor-media-placeholder {
	min-width: 48px;
    	min-height: 48px;
    	height: 48px;
}
```